blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClient.scala

```diff
@@ -13,7 +13,19 @@ import scalaz.{-\/, \/-}
 object BlazeClient {
   private[this] val logger = getLogger
 
-  def apply[A <: BlazeConnection](manager: ConnectionManager[A], idleTimeout: Duration, requestTimeout: Duration): Client = {
+  def apply[A <: BlazeConnection](manager: ConnectionManager[A], config: BlazeClientConfig): Client = {
+
+    val shutdownTask = manager.shutdown().flatMap(_ => Task.delay {
```

Comment:
Not a new problem, but still unsolved: with a managed resource, whoever creates it should be responsible for shutting it down.  We didn't create this manager.  It's similar to the executor problem.